test/pummel/test-regress-GH-814_2.js

```diff
@@ -79,14 +79,14 @@ function writer() {
         bufPool.length = 0;
       }
       process.nextTick(writer);
-      //console.error('o');
+      // console.error('o');
```

Comment:
Suggestion: remove this. If this test is going to fail, it has to be debugged one way or the other and changed. The current comment does not help at all.